components/SearchBar/index.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+import Icon from "../Icons";
+
+export default function SearchBar({ setSearchTerm }) {
+  return (
+    <StyledSearchContainer>
+      <StyledSearchForm>
+        <StyledSearchInput
+          onChange={(event) => setSearchTerm(event.target.value)}
+          type="text"
+          id="searchTerm"
+          name="searchTerm"
+          placeholder="Search Outivities..."
+        ></StyledSearchInput>
+        <Icon variant={"search"} size={25} color={"var(--primary-color)"} />
+      </StyledSearchForm>
```

Comment:
We can make `StyledSearchInput` self closing. 

```suggestion
        <StyledSearchInput
          onChange={(event) => setSearchTerm(event.target.value)}
          type="text"
          id="searchTerm"
          name="searchTerm"
          placeholder="Search Outivities..."
        />
        <Icon variant={"search"} size={25} color={"var(--primary-color)"} />
      </StyledSearchForm>
```